mypy/util.py

```diff
@@ -384,13 +467,21 @@ def style(self, text: str, color: Literal['red', 'green', 'blue', 'yellow', 'non
             start = ''
         if underline:
             start += self.UNDER
+        if dim:
+            start += self.DIM
         return start + self.colors[color] + text + self.NORMAL
 
     def colorize(self, error: str) -> str:
         """Colorize an output line by highlighting the status and error code."""
         if ': error:' in error:
             loc, msg = error.split('error:', maxsplit=1)
-            if not self.show_error_codes:
+            if self.show_source_code:
+                # Improve readability by wrapping lines when showing source code.
+                pad = len(loc) + len('error: ')
```

Comment:
This can be too high if the path the long. A fixed number such 8 might be best, or falling back to a fixed smaller number if the path is long enough (not sure which would be better).

Reply:
After some more experimentation it seems to me having zero indent here is the safest bet. Often there will be just one or few wrapped lines, and having them indented by some non-zero fixed value doesn't create a visual vertical line, so it looks a bit random. The wrapping however still makes sense so that words and types are not accidentally split.